README.md

```diff
@@ -32,8 +34,7 @@ It showcases the core ZenML concepts for supervised ML with batch predictions:
 to design flexible and reusable steps
 * using [custom data types for your artifacts and writing materializers for them](https://docs.zenml.io/user-guide/advanced-guide/artifact-management/handle-custom-data-types)
 * constructing and running a [ZenML pipeline](https://docs.zenml.io/user-guide/starter-guide/create-an-ml-pipeline)
-* accessing ZenML pipeline run artifacts in [the post-execution phase](https://docs.zenml.io/user-guide/starter-guide/fetch-runs-after-execution)
-after a pipeline run has concluded
+* TODO: add Model Control Plane docs
```

Comment:
TODO marker

Reply:
Rephrased and added a reminder to docs ticket